incubator/java-openliberty/image/project/Dockerfile

```diff
@@ -12,23 +12,29 @@ RUN cd /project && mvn -B install dependency:go-offline -DskipTests
 COPY ./util /project/util
 RUN  /project/util/check_version build
 
-# Copy the user's application pom and resolve all dependencies, also trigger Open Liberty install
+# Download Openliberty here to prevent a re-download
+RUN cd /project && mvn -B liberty:install-server liberty:create
+
+# Copy the user's application pom and resolve all dependencies
 COPY ./user-app/pom.xml /project/user-app/pom.xml
+RUN cd /project/user-app && mvn package -Dskip=true -DskipTests
```

Comment:
So the idea here is that we want 'package' to install features but not do much else?   Should we just use `liberty:install-feature` then?

Reply:
'package' will also download any other dependencies for the project, so we wont have to do this again if _just_ the app src changes

Reply:
I think this installs liberty and creates the server. So I think we want to keep it.

Reply:
Why do we need line 20 then?